pkg/cmd/cli/cmd/process.go

```diff
@@ -58,5 +100,34 @@ Examples:
 		},
 	}
 	cmd.Flags().StringP("filename", "f", "", "Filename or URL to file to use to update the resource")
+	cmd.Flags().StringP("value", "v", "", "Set value for the specified parameter")
```

Comment:
I was trying to follow docker "-e" here, if kubernetes use "-v FOO=BAR,BLA=BLAH" I can switch to that syntax